gems/pending/MiqVm/MiqRhevmVm.rb

```diff
@@ -101,12 +101,17 @@ def storage_domains_by_id
   #
   # Returns uri and mount points, hashed by storage ID.
   #
-  def add_nfs_mount(storage_obj)
+  def add_fs_mount(storage_obj)
     storage_id = storage_obj.attributes[:id]
     return if nfs_mounts[storage_id]
 
     mount_point = ::File.join(nfs_mount_root, nfs_mount_dir(storage_obj))
-    nfs_mounts[storage_id] = {:uri => "nfs://#{nfs_uri(storage_obj)}", :mount_point => mount_point, :read_only => true}
+    type = storage_obj.attributes[:storage][:type]
+    prefix = type == "nfs" ? "nfs" : "glusterfs"
+    nfs_mounts[storage_id] = {:uri => "#{prefix}://#{nfs_uri(storage_obj)}",
```

Comment:
Why do we need `prefix`? Can't we just use `type` here directly?
We know `type` can only be `nfs` or `glusterfs` based on the caller.